packages/cdk-assets/lib/private/docker.ts

```diff
@@ -0,0 +1,96 @@
+// import * as os from 'os';
+import { Logger, shell, ShellOptions } from "./shell";
+
+interface BuildOptions {
+  directory: string;
```

Comment:
```suggestion
  readonly directory: string;
```

(I know it's not required, but still)